vtm/src/org/oscim/layers/tile/vector/labeling/LabelPlacement.java

```diff
@@ -282,7 +282,12 @@ private Label addNodeLabels(MapTile t, Label l, float dx, float dy,
                     l.text.dy);
 
             for (Label o = mLabels; o != null; ) {
-                if (l.bbox.overlaps(o.bbox)) {
+                if (o.text.caption && Label.withinRepeatProximity(l, o)){
+		    o = removeLabel(o);
+                    continue;
+		}
+
+		if (l.bbox.overlaps(o.bbox)) {
```

Comment:
@gknisely Can you fix the formatting here?

You used tab for indent. We use 4 spaces.
Also an unrelated line was changed.

Reply:
no problem.  Sorry about that.